company/how-to-guides-for-staff/how-to-purchase/how-to-on-board-as-a-vendor/how-to-get-paid.md

```diff
@@ -18,23 +18,23 @@ If you are a vendor, refer to the [Bank Form](https://handbook.mattermost.com/co
 
 If you are a staff member located in Canada or if your contract is with Open World Services \(OWS\), payroll will be made to you in accordance with OWS policy. For expense reimbursement, please follow the [OWS Reimbursement Process](https://handbook.mattermost.com/company/how-to-guides-for-staff/how-to-spend-company-money/how-to-use-expensify#open-world-services-ows).
 
-#### Rest of World
+#### Rest of world
 
 If you are a vendor, or a contractor located outside the United States and Canada, Mattermost requires a primary banking institution as a method of payment, as we use Bill.com for invoice processing and payment. If you are a staff member located outside of the United States, Canada, United Kingdom and Germany, you have the option of getting paid either by bill.com (via wire transfer) or TransferWise for invoice processing, expense reimbursement and payment. The currency in which payment is made is based on the currency stated within the agreement signed with Mattermost.
 
 Primary Banking Institution and Bill.com
 
 The person or legal entity the bank account is registered to must be in the name of the contracting individual/entity.
```

Comment:
```suggestion
The person or legal entity the bank account is registered to will need to be in the name of the contracting individual/entity.
```

Reply:
@ialorro - can you please check my suggestions and make sure that you agree? If you are good, I'll commit these